template/config/webpack.test.config.js

```diff
@@ -82,20 +72,20 @@ module.exports = {
         exclude: /(node_modules)/
       },
       {
-          test: /\.(js|jsx)/,
-          enforce: 'post', // Enforce as a post step so babel can do its compliation prior to instrumenting code
-          exclude: [
-              /node_modules/,
-              /constants/,
-              /.*?(?=\.spec).*?\.js/
-          ],
-          include: project.path.src,
-          use: {
-              loader: 'istanbul-instrumenter-loader',
-              options: {
-                  esModules: true
-              }
+        test: /\.(js|jsx)/,
+        enforce: 'post', // Enforce as a post step so babel can do its compliation prior to instrumenting code
```

Comment:
compliation  -> compilation?

Reply:
loaders are not necessarily for compilation, although that's what they are normally implemented to do. A good analogy is to think of the stages in the MongoDB aggregation pipeline, where loaders would be the stages, except they may be configured to work on different data. In this case, the instrument-loader simply takes the compiled output and uses it in its stage to run the nyc cli.

Reply:
Sorry complete misunderstood what you meant. I think I might be dyslexic as I couldn't see that that word had a typo. I have updated the comment correcting the mistake.